src/OrbitGl/TrackManager.cpp

```diff
@@ -306,12 +306,23 @@ void TrackManager::AddTrack(const std::shared_ptr<Track>& track) {
   sorting_invalidated_ = true;
 }
 
+void TrackManager::AddFrameTrack(const std::shared_ptr<FrameTrack>& frame_track) {
+  frame_tracks_[frame_track->GetFunctionId()] = frame_track;
+  auto scheduler_track_pos =
+      find(sorted_tracks_.begin(), sorted_tracks_.end(), scheduler_track_.get());
+  if (scheduler_track_pos != sorted_tracks_.end()) {
+    sorted_tracks_.insert(++scheduler_track_pos, frame_track.get());
```

Comment:
maybe rename this to ordered_tracks_, I was initially under impression this is std::sorted (in which case we would have sorted this by pointers) but it is not. (not for this cl)